pinot-core/src/main/java/org/apache/pinot/core/segment/creator/impl/inv/RangeIndexCreator.java

```diff
@@ -65,7 +66,7 @@
  *   </li>
  * </ul>
  */
-public final class RangeIndexCreator implements InvertedIndexCreator {
+public final class RangeIndexCreator implements RawValueBasedInvertedIndexCreator {
```

Comment:
This should implement both `RawValueBasedInvertedIndexCreator` and `DictionaryBasedInvertedIndexCreator`?

Reply:
done